### PR TITLE
Add -UseBasicParsing to create issue script

### PR DESCRIPTION
**Pull request by michalb-sonar**

### CODE REVIEW

File: sonaranalyzer-dotnet/build/create-issue.ps1 [hunk 1]
```diff
@@ -100,7 +100,8 @@ function New-Issue() {
         -Uri $projectUri `
         -Method "POST" `
         -Headers $headers `
-        -Body ($payload | ConvertTo-Json)
+        -Body ($payload | ConvertTo-Json) `
+        -UseBasicParsing
```
Comment (Evangelink) [inline sonaranalyzer-dotnet/build/create-issue.ps1]: What's the goal of this change?

Comment (michalb-sonar) [inline sonaranalyzer-dotnet/build/create-issue.ps1]: It allows the script to be used on machines where IE is not installed or where IE initial configuration is not performed.

Description from msdn:
"-UseBasicParsing Indicates that the cmdlet uses the response object for HTML content without Document Object Model (DOM) parsing."
We do not need to parse the DOM in scripts.

Comment (Evangelink) [inline sonaranalyzer-dotnet/build/create-issue.ps1]: Yep I read the documentation I was just curious to know what was the motivation behind this change.

**Comment by Evangelink** [comment]
rebase done

**Pull request merged**